packages/aws-rfdk/lib/deadline/README.md

```diff
@@ -163,6 +190,57 @@ const renderQueue = new RenderQueue(stack, 'RenderQueue', {
 });
 ```
 
+### Render Queue Subnet Placement
+
+We highly recommend creating dedicated subnets for the Render Queue's [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html)
+that will only be used by the load balancer. Using dedicated subnets for load balancers is considered best practice and is especially important if you are using
+Deadline Secrets Management (see [Deadline Secrets Management Considerations](#using-dedicated-subnets-for-deadline-components)).
+
+The following example creates a dedicated subnet group for the `RenderQueue` to use for its Application Load balancer:
+```ts
+const vpc = new Vpc(this, 'Vpc', {
+  // ...
+  subnetConfiguration: [
+    // ...
+
+    // Provide a subnet configuration for the Render Queue subnet group
+    {
+      name: 'RenderQueueSubnets',
+      subnetType: SubnetType.PRIVATE,
+      cidrMask: 27,
+    },
+
+    // ...
+  ],
+  // ...
+});
+
+// ...
+
+const renderQueue = new RenderQueue(stack, 'RenderQueue', {
+  // ...
+  vpc,
+
+  // Select the dedicated Render Queue subnets to put the ALB in
+  vpcSubnetsAlb: vpc.selectSubnets({ subnetGroupName: 'RenderQueueSubnets' }),
+
+  // ...
+});
+```
+
+Application Load Balancers have requirements for the subnets they are deployed into, such as requiring that each subnet be from a different Availability Zone and that each subnet
+have a CIDR block with at least a `/27` bitmask and at least 8 free IP addresses per subnet. For the full list of requirements, please see the
+[Elastic Load Balancing documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#subnets-load-balancer).
+
+### Configuring Deadline Secrets Management on the Render Queue
+
+When [Deadline Secrets Management](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html) is enabled on the `Repository` construct,
+the `RenderQueue` will automatically configure itself as a [Server role](https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/secrets-management/deadline-secrets-management.html#assigned-roles) in Deadline Secrets Management.
+
+TODO: Add more details about auto-registration rules and how to use them (also in dev guide)
```

Comment:
RFDK will handle configuring "auto-registration rules" for client constructs (`WorkerInstanceFleet`, `SpotEventPluginFleet`, and `UsageBasedLicensing`). In terms of documentation, users only need to be aware that the subnets chosen for those constructs should be dedicated. Perhaps that can be covered by the suggested sub-sections comment above and we can remove this `TODO` here.

What do you think?

Reply:
Yeah that sounds good to me, removed.